packages/server/template.html

```diff
@@ -1,6 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <!-- Google Tag Manager -->
+    <script>
```

Comment:
wasn't there also a script for the body? curious what happened to that?

Reply:
Good Q. Looking into it more, the body tag is only used for clients who don't use javascript. Since that point is moot with us, I decided to nuke it

Reply:
(the way it worked was it loaded a page in an iframe, I presume that Google runs the javascript on their server and returns the rendered page